internal/controller/cloud-control/vpcpeering_aws_test.go

```diff
@@ -63,6 +73,14 @@ var _ = Describe("Feature: KCP VpcPeering", func() {
 				).
 				Should(Succeed())
 		})
+
+		By("And Then KCP VpcPeering has status.vpcId equal to existing AWS VPC id", func() {
+			Expect(vpcpeering.Status.VpcId).To(Equal(vpcId))
+		})
+
+		By("And Then KCP VpcPeering has status.ConnectionId equal to existing AWS Connection id", func() {
+			Expect(vpcpeering.Status.ConnectionId).To(Equal(connectionId))
```

Comment:
Shouldn't it be listed from the mock and checked if mock value is equal to the status value, rather then be defined so hard-coded in the test?

Reply:
I agree. I will use values from mock.